jetstream/objectstore.ts

```diff
@@ -214,22 +214,11 @@ export class ObjectStoreImpl implements ObjectStore {
     this.js = js;
   }
 
-  _sanitizeName(name: string): { name: string; error?: Error } {
+  _checkNotEmpty(name: string): { name: string; error?: Error } {
```

Comment:
Do you think this check needs to return the name?
Wouldn't accepting the current one and returning bool for empty or not be enough?

Reply:
then I have to wrap for a try-catch, the name is retuned in the object